generate.sh

```diff
@@ -42,21 +33,25 @@ usage() {
     echo "  --pkgCsv <pathToCSV>               A path to a csv file of packages to generate. Format is the same as the --pkg"
     echo "                                       option above, one per line.  If specified, the --pkg option is ignored."
     echo "  --dest <pathToDestRepo>            A path to the root of the repo to copy source into."
+    echo "  --type <packageType>               Type of the package to generate. Accepted values: text | ref."
```

Comment:
The ending punctuation is confusing IMO especially when it is included the the next line that indicates the default.  It can leave the user with the impression the value they must specify is actually `ref.`
```suggestion
    echo "  --type <packageType>               Type of the package to generate. Accepted values: ref (default) | text"
```

Reply:
Sure, thanks for pointing it out, will change